src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -1602,54 +1595,54 @@ a.tribe-rating-link {
 Red button styles.
 
 Credit to https://github.com/jensjns/wp-admin-red-button
-=============================================*/
+============================================= */
 .wp-core-ui .button-red {
 	background-color: #a00;
-	border-bottom-color: #8D1F21;
-	border-color: #9B2124;
-	box-shadow: inset 0 1px 0 rgba(120,200,230,0.5);
+	border-color: #9b2124;
+	border-bottom-color: #8d1f21;
+	box-shadow: inset 0 1px 0 rgba(120, 200, 230, 0.5);
```

Comment:
With the previous order, the `border-color` will overwrite the `border-bottom-color`. So in the fixed version, we should really only need `border-color` and remove the `border-bottom-color`, as it doesn't take effect.